dts/bindings/vendor-prefixes.txt

```diff
@@ -456,6 +456,7 @@ xunlong	Shenzhen Xunlong Software CO.,Limited
 ysoft	Y Soft Corporation a.s.
 zarlink	Zarlink Semiconductor
 zeitec	ZEITEC Semiconductor Co., LTD.
+zephyr	Zephyr
```

Comment:
Can you make this:

`zephyr zephyr-specific binding`

Reply:
Sure! This has been updated